integration/single/test_api_with_domain_ipaddresstype.py

```diff
@@ -0,0 +1,35 @@
+from unittest.case import skipIf
+
+from integration.config.service_names import REST_API
+from integration.helpers.base_internal_test import BaseInternalTest
+from integration.helpers.resource import current_region_does_not_support
+
+
+@skipIf(
+    current_region_does_not_support([REST_API]),
```

Comment:
Please change the check from `REST_API` to `CUSTOM_DOMAIN` (`CUSTOM_DOMAIN` is more restrictive), an example here (we need to change the `current_region_does_not_support()` method to `current_region_not_included()` too): https://github.com/aws/serverless-application-model/blob/893aa90cd2a0b0ca1220108342bc9398d89cf4e7/integration/combination/test_custom_rest_api_domains.py#L10